### PR TITLE
fix: patch scroll of homepage on mobile

### PR DESCRIPTION
**Pull request by Keith-CY**
Before

https://user-images.githubusercontent.com/7271329/227099672-cb0dfad3-3d61-43c1-a554-a20f65f17683.mov

After

https://user-images.githubusercontent.com/7271329/227099705-03ac78bc-646c-423c-b581-447c65f69bd6.mov

### CODE REVIEW

**Comment by Keith-CY** [comment]
This is a patch of the scroll behavior on the mobile layout, could be dismissed if there's a better solution

File: src/pages/home/index.page.tsx [hunk 1]
```diff
@@ -245,6 +245,77 @@ const SlideCKBModular: FC<ScreenSlideProps> = props => {
 allowCustomDescendantOfSwiper(SlideCKBModular)
 
 const SlideGetStarted: FC<ScreenSlideProps> = props => {
+  const isMobile = useIsMobile()
+  if (isMobile) {
+    return (
```
Comment (WhiteMinds) [inline src/pages/home/index.page.tsx:250]:
```suggestion
  const isMobile = useIsMobile()

  if (isMobile) {
    // TODO: This is a temporary solution to the problem of not being able to
    // view the full content of this slideshow under mobile.
    // In the future, it is expected that the swiper's freeMode.sticky property
    // will be used to achieve better results, but some hacking may be required.
    return (
```

File: src/pages/home/index.page.tsx [hunk 2]
```diff
@@ -245,6 +245,77 @@ const SlideCKBModular: FC<ScreenSlideProps> = props => {
...
+      <ScreenSlide autoHeight {...props} className={clsx(presets.themeDark, props.className)}>
+        <div className={styles.slideGetStarted}>
+          <div className={clsx(styles.titleText, DISABLE_CGOL_MOUSE_CONTROLLER)}>Get Started.</div>
+          <div className={styles.cards}>
+            <Swiper
+              direction="vertical"
+              slidesPerView="auto"
+              // autoHeight
```
Comment (WhiteMinds) [inline src/pages/home/index.page.tsx:258]:
```suggestion
```

**Review by WhiteMinds — CHANGES_REQUESTED** [review]

**Comment by Keith-CY** [comment]
Suggestions are committed, please have a review @WhiteMinds

**Review by WhiteMinds — APPROVED** [review]

**Pull request merged**